### DISCUSSION TITLE
Is the UWP ColorPaletteResources Class not available/not functional in Winui2.x?

### DISCUSSION THREAD

**Discussion by groovykool** [microsoft/microsoft-ui-xaml#8922] [Q&A] [answered]
Is the UWP ColorPaletteResources Class not available/not functional in Winui2.x?

**Comment by marcelwgn** [comment, marked as answer]
WinUI 2 is a component library for the UWP Platform, so the UWP ColorPaletteResources class only applies to UWP. From my knowledge, the class did not exist in WinUI 2 and since Fluent Design also moved to a new brush system, the class is probably also not up to date with current design guidance.
